src/paillier.rs

```diff
@@ -219,8 +219,8 @@ pub(crate) mod prime_gen {
     use rand::{CryptoRng, RngCore};
 
     /// Sample a safe prime with length `PRIME_BITS` at random.
-    pub(crate) fn get_random_safe_prime<R: RngCore + CryptoRng>(rng: &mut R) -> BigNumber {
-        BigNumber::safe_prime_from_rng(PRIME_BITS, rng)
+    pub(crate) fn get_random_safe_prime<R: RngCore + CryptoRng>(_rng: &mut R) -> BigNumber {
+        BigNumber::safe_prime(PRIME_BITS)
```

Comment:
thought: I don't love that OpenSSL forces us to use an unspecified RNG. I'm inclined to merge this ASAP so everything else can iterate faster, but this bums me out.

Reply:
Oh yeah, I meant to comment on this line and ask your opinion. OpenSSL doesn't have an equivalent function. Is this okay? I'm not sure what an alternative would be </3

Reply:
I left [a comment on the issue](https://github.com/boltlabs-inc/tss-ecdsa/issues/96#issuecomment-1398538283) about it, but as far as I can tell, the solution is to use gmp instead of OpenSSL. I don't know why ossl allows RNG specification for some methods but not for this one.

@amaloz do you have thoughts on this?

Reply:
After discussion, we have switched to the GMP implementation. Which is also already installed on the GitHub Action Ubuntu instance.